chainbase/src/main/java/org/tron/core/state/WorldStateGenesis.java

```diff
@@ -177,13 +186,26 @@ private void initGenesisDBs() {
     Arrays.stream(StateType.values()).filter(type -> type != StateType.UNDEFINED)
         .parallel().forEach(type -> {
           try {
-            genesisDBs.put(type, new RocksDB(stateGenesisPath, type.getName()));
-          } catch (RocksDBException e) {
+            genesisDBs.put(type, getDb(stateGenesisPath, type.getName()));
+          } catch (RocksDBException | IOException e) {
             throw new RuntimeException(e);
           }
         });
   }
 
+  private DB getDb(Path sourceDir, String dbName) throws RocksDBException, IOException {
+    File engineFile = Paths.get(sourceDir.toString(), dbName, ENGINE_FILE).toFile();
+    if (!engineFile.exists()) {
+      return  new RocksDB(sourceDir, dbName);
```

Comment:
If the file does not exist, is it a LevelDB?

Reply:
yes